pkg/jx/cmd/preview.go

```diff
@@ -105,6 +105,10 @@ func NewCmdPreview(f Factory, in terminal.FileReader, out terminal.FileWriter, e
 		Run: func(cmd *cobra.Command, args []string) {
 			options.Cmd = cmd
 			options.Args = args
+			//Default to batch-mode when running inside the pipeline (but user override wins).
+			if !cmd.Flag("batch-mode").Changed {
```

Comment:
Having added optionBatchMode as a constant to `common.go` it would make sense to use it here (and also ask people to start using it) - I've been wanting to make this change for a while!

Otherwise 👍

Reply:
D'oh! That is _totally_ what I meant to do and the reason I made them all constants. I just managed to miss out the other half of it! :-(